docs/faq.md

```diff
@@ -167,3 +173,9 @@ sudo systemctl restart ltapi
 ```
 
 You will have to run this for every loadtest agent you have. These will be appended by `agent-` when you run the `ltctl ssh` command above.
+
+### What's the purpose of the `ServerURL` and `ServerScheme` settings?
+
+This are intended for users that need to override the connection URL to their Mattermost server in their load tests environments, in most cases because there's a custom reverse proxy in front of the load-test deployment.
```

Comment:
Typo:
```suggestion
These are intended for users that need to override the connection URL to their Mattermost server in their load tests environments, in most cases because there's a custom reverse proxy in front of the load-test deployment.
```